vllm/engine/arg_utils.py

```diff
@@ -247,11 +247,13 @@ def _compute_kwargs(cls: ConfigType) -> dict[str, dict[str, Any]]:
             default = field.default
             # Handle pydantic.Field defaults
             if isinstance(default, FieldInfo):
-                default = (
-                    default.default
-                    if default.default_factory is None
-                    else default.default_factory()
-                )
+                if default.default_factory is None:
+                    default = default.default
+                else:
+                    # A default vllm_config will be constructed here.
+                    # Info logs will be output, which is confusing.
```

Comment:
I don't want to waste CI, but I'd update this comment as it's not technically correct

```suggestion
                    # VllmConfig's Fields have default_factory set to config classes.
                    # These could emit logs on init, which would be confusing.
```